scripts/fork/usdtDeposit.js

```diff
@@ -0,0 +1,169 @@
+const hardhat = require('hardhat')
+const toWei = ethers.utils.parseEther
+
+async function getEvents(contract, tx) {
+    let receipt = await ethers.provider.getTransactionReceipt(tx.hash);
+    return receipt.logs.reduce((parsedEvents, log) => {
+        try {
+            parsedEvents.push(contract.interface.parseLog(log));
+        } catch (e) { }
+        return parsedEvents;
+    }, []);
+}
+
+async function run() {
+    const overrides = { gasLimit: 9500000 }
+    let wallet, wallet2, wallet3, wallet4
+    let usdt
+    let prizePool
+    let poolWithMultipleWinnersBuilder;
+    let compoundPrizePoolABI;
+
+    [wallet, wallet2, wallet3, wallet4] = await hardhat.ethers.getSigners()
+
+    usdt = await ethers.getContractAt('TetherToken', '0xdAC17F958D2ee523a2206206994597C13D831ec7')
```

Comment:
It appears that you only use the `approve()` and `transfer()` functions for the USDT token.  Those are both part of ERC20, so you could reuse the standard ERC20 abi:

```javascript
usdt = await ethers.getContractAt('IERC20Upgradeable', '0xdAC17F958D2ee523a2206206994597C13D831ec7')
```

And then you can remove the `TetherToken.sol`.